hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/support/validators/block/StateChangesValidator.java

```diff
@@ -145,13 +146,15 @@ public class StateChangesValidator implements BlockStreamValidator {
     public static void main(String[] args) {
         final var validator = new StateChangesValidator(
                 Bytes.fromHex(
-                        "a1d57e05d0607f76d13bc2ed31c40585adc294927e767df1f3443f065cc7a3348d3e0c4dd5581e6922c718e16b787231"),
+                        "d7d0cb025b3c5a4043b7c8d801b13caf49d4ba8f9b486d2fc09ec6a3cfe3d9743a659582e3d953ad4bc23cae9c87c628"),
                 Paths.get(
-                        "/Users/michaeltinker/AlsoDev/hedera-services/hedera-node/test-clients/build/hapi-test/node0/output/swirlds.log"),
+                        "/Users/michaeltinker/YetAnotherDev/hedera-services/hedera-node/test-clients/build/hapi-test/node0/output/swirlds.log"),
                 Paths.get(
-                        "/Users/michaeltinker/AlsoDev/hedera-services/hedera-node/test-clients/build/hapi-test/node0/config.txt"));
+                        "/Users/michaeltinker/YetAnotherDev/hedera-services/hedera-node/test-clients/build/hapi-test/node0/config.txt"),
+                Paths.get(
+                        "/Users/michaeltinker/YetAnotherDev/hedera-services/hedera-node/test-clients/build/hapi-test/node0/data/config/application.properties"));
         final var input =
-                "/Users/michaeltinker/AlsoDev/hedera-services/hedera-node/test-clients/build/hapi-test/node0/data/block-streams/block-0.0.3/";
+                "/Users/michaeltinker/YetAnotherDev/hedera-services/hedera-node/test-clients/build/hapi-test/node0/data/block-streams/block-0.0.3/";
```

Comment:
All of the strings here could be made less sensitive (so we don't have to change the code for each different engineer or clone location) by replacing the node data folder prefix `"/Users/michaeltinker/YetAnotherDev/hedera-services/hedera-node/test-clients/build/hapi-test/node0/"` with the following code.
```java
        final Path nodeZeroBaseFolder =
                WorkingDirUtils.workingDirFor(0, "hapi").toAbsolutePath().normalize();
```

Reply:
Done.